README.md

```diff
@@ -155,13 +205,13 @@ Example usage:
 
 ## Coverage (internal)
 
-This action should not probably used by workflows, rather by other actions, that implement unit tests for given languages. The action allows storing coverge of given branch in AWS S3 and comparing coverage results during PRs. It ensures, that coverage is incrementally improved with time, by enforcing small improvement with every PR, up to given threshold (currently 80%).
+This action should not probably used directly by workflows, rather by other actions, that implement unit tests for given languages. The action allows storing coverge of given branch in AWS S3 and comparing coverage results during PRs. It ensures, that coverage is incrementally improved with time, by enforcing small improvement with every PR, up to given threshold (currently 80%).
```

Comment:
`with time` -> `over time`

Fortunately it doesn't really matter. Approve time ^

Reply:
Thanks, fixed